actionmailer/lib/action_mailer/queued_delivery.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActionMailer
+  module QueuedDelivery
+    extend ActiveSupport::Concern
+
+    included do
+      class_attribute :delivery_job, default: ::ActionMailer::MailDeliveryJob
+      class_attribute :deliver_later_queue_name, default: :mailers
```

Comment:
@packagethief shouldn't this be `default: -> { ActionMailer::Base.deliver_later_queue_name }`? Since Rails 6.1 mailers queue is nil / :default instead of :mailers.

Reply:
This is defining `ActionMailer::Base.deliver_later_queue_name` so it can't have a default of itself.

Good catch though @javierav. I'd missed that in the 6.1+ configuration defaults, the value is being set to nil:

https://github.com/rails/rails/blob/c15093c8561618a7673d1373293e6ff58a05cd65/railties/lib/rails/application/configuration.rb#L212-L214

The effect is the same – we changed from a `cattr_accessor` to a `class_attribute`, but both had the same default of `:mailers`. In configurations < 6.1 this will be respected, and in later configurations nillified.

But it means the documentation I added about the `:mailers` name in this PR is wrong, since that won't be the default in newly-generated configurations. I'll create a new PR to update that 👍

Reply:
Updated the documentation in https://github.com/rails/rails/pull/47519